lnst/Recipes/ENRT/ConfigMixins/EnableIntelSSTTFMixin.py

```diff
@@ -0,0 +1,110 @@
+import re
+
+from lnst.Common.LnstError import LnstError
+from lnst.Common.Parameters import BoolParam
+from lnst.Controller.RecipeResults import ResultLevel, ResultType
+from lnst.Recipes.ENRT.ConfigMixins import BaseSubConfigMixin
+
+
+class EnableIntelSSTTFMixin(BaseSubConfigMixin):
+    enable_intel_sst_tf = BoolParam(default=False)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if getattr(self.params, "disable_turboboost", True):
```

Comment:
It's probably not a big deal, but could we make this check not dependent on the other mixin that does not have to be in other Recipe instance hierarchy?

I'd check the value of `/sys/devices/system/cpu/intel_pstate/no_turbo` instead.